tgui/packages/tgui/interfaces/ReligiousTool.js

```diff
@@ -0,0 +1,217 @@
+import { capitalize } from 'common/string';
+import { useBackend, useSharedState } from '../backend';
+import { AnimatedNumber, BlockQuote, Box, Button, Collapsible, Dimmer, Icon, LabeledList, NoticeBox, ProgressBar, Section, Stack, Tabs } from '../components';
+import { Window } from '../layouts';
+
+const ALIGNMENT2COLOR = {
+  "good": "yellow",
+  "neutral": "white",
+  "evil": "red",
+};
+
+export const ReligiousTool = (props, context) => {
+  const { act, data } = useBackend(context);
+  const [tab, setTab] = useSharedState(context, 'tab', 1);
+  const {
+    sects,
+    alignment,
+    toolname,
+  } = data;
+  return (
+    <Window
+      title={toolname}
+      width={560}
+      height={500}>
+      <Window.Content scrollable>
+        <Stack vertical fill>
```

Comment:
Fill probably does not work with scrollable stuff, because there is no defined height.

Reply:
It's worth double checking, let me look into that

Reply:
Anyway, just keep it in mind, I'm pressing the green button.